lnwallet/interface.go

```diff
@@ -124,6 +124,9 @@ type TransactionDetail struct {
 
 	// Label is an optional transaction label.
 	Label string
+
+	// PreviousOutPoints are the inputs for a transaction
```

Comment:
nit: all comments should end with a full stop. Also we should try to be consistent with the naming/capitalization, so it should be `PreviousOutpoints` (instead of `PreviousOutPoints`) everywhere.